src/udisksspawnedjob.c

```diff
@@ -1032,6 +1032,8 @@ void udisks_spawned_job_start (UDisksSpawnedJob *job)
     }
 
   job->child_stdout_channel = g_io_channel_unix_new (job->child_stdout_fd);
+  // we want to read binary, suppress checking the encoding:
```

Comment:
Please use C-style comments.

Reply:
Done. (I was copying the style from the corresponding `set_encoding()` call for stdin, above.)

Reply:
Sigh, will schedule a clean sweep round soon. Thanks for pointing onto that.